help.html

```diff
@@ -88,27 +88,30 @@ <h1 id="getting-help-with-astropy">Getting Help with Astropy<a class="paralink"
 		please message one of the <a
 		href="https://www.facebook.com/groups/astropython/admins/">admins</a> with a
 		brief description of your involvement in astronomy. This is a very active
-		group and the best place for beginners/learners to get help - most posts get
-		responses within a few hours.</li>
+		group and is the best place to get help. Most posts get responses within a few hours.
+		</li>
 
 		<li><a href="http://mail.python.org/mailman/listinfo/astropy">Users Email
 		List</a> - Post questions or start discussions about anything related to
 		Python programming applied to astronomy. This is an active email list and
 		most posts get a response within a few days.</li>
 
-		<li> <a href="https://astropy.slack.com">Astropy Slack team</a> (get an account <a
-		href="http://joinslack.astropy.org">here</a>)- A number of developers and
-		contributors are using Slack (which has a live web chat format)
-		for everyday discussions, so please do feel free to drop by if you have
-		questions about using or contributing to astropy. Be aware that while this
-		is a live chat forum, depending on the time of day/year there may
-		not necessarily be someone available to help, so if you don't get an answer
-		here, be sure to try out one of the other methods of communication!</li>
+		<li> <a href="https://astropy.slack.com">Astropy Slack team</a> (<a
+		href="http://joinslack.astropy.org">get an account</a>) - Some developers and
+		contributors using Slack for everyday discussions. Feel free to use this
+		forum if you have any questions about using or contributing to astropy.
+		Please browse all the available channels before posting and tro to choose the most
+		relevant one.
+		</li>
 
 		<li> <a href="http://groups.google.com/group/astropy-dev">Developer Email
-		List</a> - Start discussion and ask questions about changing Astropy's
+		List [astropy-dev]</a> - Start discussion and ask questions about changing astropy's
```

Comment:
Don't we typically capitalize if we mean "Astropy" the project, as opposed to "astropy" the package?

Reply:
Yes. Capitalization matters in terms of branding consistency.

Reply:
To reflect @hamogu's suggestion, this would be:
```suggestion
		List [astropy-dev]</a> - Start discussion and ask questions about changing Astropy's
```

Reply:
I do usually lowercase astropy when I mean the package and try to spell out and capitalize Astropy Project. In this case, I meant the code, so I used lowercase. Is the sentence ambiguous? I'll try to rewrite.

Reply:
I rewrote the sentence and used the <code> tag to make it even more clear.

Reply:
Specifically, `astropy` only means the *core* package. It doesn't include affiliated or coordinated. That is my understanding anyway.